src/tools/pbs_probe.c

```diff
@@ -1527,8 +1527,8 @@ adjust_for_os(struct infrastruct *pinf)
 		/* Linux + /etc/sgi-compute-node_release => SGI ICE	*/
 		if (access("/etc/sgi-compute-node-release", R_OK) == 0) {
 			lib_mpugs[ofs_lib + 23].notReq = 0;    /* sgiMPI.awk       */
-			sbin_mpugs[ofs_sbin + 5].notReq = 0x2; /* pbs_mom.cpuset   */
-			sbin_mpugs[ofs_sbin + 6].notReq = 0x2; /* pbs_mom.standard */
+			sbin_mpugs[ofs_sbin + 5].notReq = 0x1; /* pbs_mom.cpuset   */
+			sbin_mpugs[ofs_sbin + 6].notReq = 0x1; /* pbs_mom.standard */
```

Comment:
I think we should simply remove these two lines. If the now unused slots are reused in the future, this could cause a problem.